docs/tutorials/building-your-first-bot.mdx

```diff
@@ -0,0 +1,346 @@
+
+Feel free to follow this tutorial within your own game, but we recommend starting with a simple sample game. Clone the project
+from the [Regression-Games-Marketplace/SampleDemoScene](https://github.com/Regression-Games-Marketplace/SampleDemoScene.git) GitHub repository.
+
+```
+git clone https://github.com/Regression-Games-Marketplace/SampleDemoScene.git
+```
+
+Open it within Unity version 2021.3.24f1 (visit the [Unity archive](https://unity.com/releases/editor/whats-new/2021.3.24) if 
+you can't find that version). Once the project is loaded, a default scene with bots in a playground should open. If it does not, 
+then find the **Playground** scene under **Assets > FirstBotDemo > Demo > Scenes** and open it manually. If you'd like, you can 
+click play now to see how the characters move.
+
+![Screenshot of the playground scene](./img/building-your-first-bot/start_0.png)
+
+## Import the Regression Games Unity Bots package
+
+You can find [our package on GitHub](https://github.com/Regression-Games/RGUnityBots). Import the package into your Unity project by opening 
+the **Package Manager** window (`Window > Package Manager`) and click **Add package from git URL**. Then, paste the following URL:
+
+```
+https://github.com/Regression-Games/RGUnityBots.git#v0.0.17
+```
+
+:::info
+
+- This package utilizes TextMeshPro.  If you are prompted by Unity to add TextMeshPro assets to your project, please add them.
+- This package installs a dependency of Newtonsoft Json (https://docs.unity3d.com/Packages/com.unity.nuget.newtonsoft-json@3.1/manual/index.html) for serializing/deserializing Json message payloads.
+- If you get a note about Unity's new input system, click **Yes** and allow the editor to restart.
+
+:::
+
+![Screenshot of the package imported into the project](./img/building-your-first-bot/welcome_3.png)
+
+After importing the package, you will be greeted by a login screen. Create an account at
+[https://play.regression.gg](https://play.regression.gg) and login to Unity with your account credentials. If you need to login again in the future, 
+you can enter your login info by visiting the **Edit** > **Project Settings** > **Regression Games** menu.
+
+## Restart Unity and your IDE
+
+After installing the Regression Games Unity Bots package and its dependencies into Unity, please restart Unity 
+and your code editor (e.g. Rider or Visual Studio). Without this step, Unity and your code editor may not properly 
+resolve the new packages even after Unity recompiles the scripts.
+
+## Add the RGOverlayCanvas
+
+The RGOverlayCanvas prefab provides a drag and drop overlay that lets you easily start and stop bots running
+in your scene. This is extremely useful when initially creating and debugging your bots.
+
+:::caution
+
+Note that this overlay **must be present in your scene** for Regression Games to function. The visibility
+of the overlay can be hidden through the settings. If you have multiple scenes where a bot will be used,
```

Comment:
```suggestion
of the overlay can be hidden through its settings. If you have multiple scenes where a bot will be used,
```

Reply:
I think it should be `the`? It's not the overlay's settings - it's the settings of the RG SDK.

Reply:
I'd make that explicit then - I thought this was talking about hiding the component from its GameObject. If there's another setting then we should mention where that is